### ISSUE TITLE
Focus after selection - keyboard navigation broken

### ISSUE THREAD

**Issue by rimblas** [nbuytaert1/apex-select2#71]
On the UT, after a selection is made, focus is not left on the Select2 item, therefore it's not possible to tab to the next field on the page.
After a value is selected, focus should remain on the item. Select2 3.5.3 seems to behave this way already.

**Comment by nbuytaert1** [comment]
That's a bug in the Select2 jQuery plugin.
https://github.com/select2/select2/issues/4384
Happens only for single-value page items.

Fixed by adding the following IF condition:

```sql
if l_select_list_type = 'SINGLE' then
  l_code := l_code || '
    $("' || l_item_jq || '").on(
      "select2:select",
      function(){ $(this).focus(); }
    );';
end if;
```

Thanks for the bug notification, Jorge.

**Comment by nbuytaert1** [comment]
Fixed in release version 3.0.3.
